### ISSUE TITLE
Add ability to not use Author Folder Format for renaming

### ISSUE THREAD

**Issue by goaliedude3919** [Readarr/Readarr#3427]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Not really a problem, per se.

### Describe the solution you'd like

I do not want to have my books organized by author. I would prefer for all my books to simply be in one main folder. As such, it would be nice to add an option for {None} or {Ignore} or something like that for the Author Folder Format.

### Describe alternatives you've considered

There's not really an alternative because the Author Format Folder configuration requires one of the inflation tokens to be entered and does not allow a simple hard-coded folder name.

### Anything else?

N/A

**Comment by drysdan** [comment]
Found this issue when searching for how to enable this (apparently missing) feature. I also want to have all my books in one main folder, with only book subfolders. Apparently this is currently impossible, but I would like to add my voice to the request to implement this option

**Comment by jdestef90** [comment]
I would also like to add my voice to this request. Specifically, when using AudioBookShelf the scanner does not recognize author folders, it assumes any folder within the parent library folder is a book which makes importing the library very difficult.

**Comment by michaudtime** [comment]
Yes this please! Sort by author is cool but it would be nice to be able to have it for a book per sub-folder in my root Books/ Folder

**Comment by CounterClops** [comment]
Yeah, I'd like to second this as well. I asked about this in the Readarr Discord back in 2023, and was told to find something else instead. I would love to use Readarr, but right now the fact the author folder is a requirement stops me from being able to use it.

My core example, was because I wanted to organise my Warhammer 40k ebooks/audiobooks, where a single series can flip between various authors between each book.
